views/newEvent.handlebars

```diff
@@ -1,112 +1,142 @@
-                      <option value="OR">Oregon</option>
-                      <option value="PA">Pennsylvania</option>
-                      <option value="PR">Puerto Rico</option>
-                      <option value="RI">Rhode Island</option>
-                      <option value="SC">South Carolina</option>
-                      <option value="SD">South Dakota</option>
-                      <option value="TN">Tennessee</option>
-                      <option value="TX">Texas</option>
-                      <option value="UT">Utah</option>
-                      <option value="VA">Virginia</option>
-                      <option value="VT">Vermont</option>
-                      <option value="WA">Washington</option>
-                      <option value="WI">Wisconsin</option>
-                      <option value="WV">West Virginia</option>
-                      <option value="WY">Wyoming</option> 
-                    </select>
-                  </div>
-                  <div class="form-group col-md-2">
-                    <label for="inputZip">Zip</label>
-                    <input type="text" class="form-control" id="inputZip">
-                  </div>
-                </div>
-                <div class="form-row mt-3">
-                    <div class="form-group col-md-6">
-                        <div id="date-picker-start" class="md-form md-outline input-with-post-icon datepicker">
-                            <label for="example">Event Start Date...</label>
-                            <input placeholder="Select start date" type="text" id="eventStart" class="form-control">
-                            <i class="fa fa-calendar input-prefix" tabindex=0></i>
-                          </div>
-                    </div>
-                    <div class="form-group col-md-6">
-                        <div id="date-picker-end" class="md-form md-outline input-with-post-icon datepicker">
-                            <label for="example">Event End Date...</label>
-                            <input placeholder="Select end date" type="text" id="eventEnd" class="form-control">
-                            <i class="fa fa-calendar input-prefix" tabindex=0></i>
-                          </div>
-                    </div>
+            </div>
+        </form>            
+    </div>
 
-                </div>
-                <button type="submit" class="btn btn-primary">Create Event</button>
-            </form>            
+
+    <!-- Add event tasks -->
+    <div class="row mb-3 justify-content-center">
+      <h2>Add Volunteer Tasks</h2>
+    </div>
+
+    <form class="justify-content-center d-flex mb-3">
+      <div class="input-group col-md-6 mb-3">
+        <input type="text" class="form-control" placeholder="Add volunteer task..." aria-label="Volunteer task" aria-describedby="button-addon2">
+        <div class="input-group-append">
+          <button class="btn btn-outline-light" type="button" id="button-addon2">Add Task</button>
```

Comment:
id should be added to match the addTask.js

Reply:
Note to match IDs